docs/server/source/cloud-deployment-templates/add-node-on-kubernetes.rst

```diff
@@ -161,3 +161,14 @@ zero downtime during updates.
 
 You can SSH to an existing BigchainDB instance and run the ``bigchaindb
 show-config`` command to check that the keyring is updated.
+
+
+Step 7: Run NGINX as a Deployment - Optional
+--------------------------------------------
+
+This step is required only if you are want to control access to the BigchainDB
```

Comment:
It says Step 7 is optional, but the updated YAML files for the MongoDB Service and the BigchainDB Service make them so they are no longer `type: LoadBalancer`, i.e. there is an implicit assumption of NGINX (or something else) between those Service endpoints and the public Internet now, so this step is not optional.

Option 1: Make this step non-optional.

Option 2: Add documentation to change
```text
type: ClusterIP
clusterIP: None
```
to `type: LoadBalancer` in the Service definition YAML files for BigchainDB and MongoDB.

I prefer Option 1.